libs/local_lcos/include/hpx/local_lcos/force_linking.hpp

```diff
@@ -5,10 +5,10 @@
 //  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
 
 #if !defined(HPX_LOCAL_LCOS_FORCE_LINKING_HPP)
```

Comment:
```suggestion
#ifndef HPX_LOCAL_LCOS_FORCE_LINKING_HPP
```
?

Reply:
clang-format recognizes the `#ifndef` as #include guards, however it does not recognize the `#if !defined()` version as such.